code/game/objects/items/kitchen.dm

```diff
@@ -276,15 +276,33 @@
 	return BRUTELOSS
 /* Trays  moved to /obj/item/storage/bag */
 
+/obj/item/kitchen/spoon
+	name = "spoon"
+	desc = "Just be careful your food doesn't melt the spoon first."
+	icon_state = "plastic_spoon"
+	w_class = WEIGHT_CLASS_TINY
+	flags_1 = CONDUCT_1
+	force = 2
+	throw_speed = 3
+	throw_range = 5
+	attack_verb_simple = list("whack", "spoon", "tap")
+	attack_verb_continuous = list("whacks", "spoons", "taps")
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, RAD = 0, FIRE = 50, ACID = 30)
+	custom_materials = list(/datum/material/iron=120)
+	custom_price = PAYCHECK_PRISONER * 5
+	tool_behaviour = TOOL_MINING
+	toolspeed = 25 // Literally 25 times worse than the base pickaxe
+
 /obj/item/kitchen/spoon/plastic
 	name = "plastic spoon"
 	desc = "Just be careful your food doesn't melt the spoon first."
 	icon_state = "plastic_spoon"
```

Comment:
remove some of these since they're duped now

Reply:
Done